tests/storage/hotplug.go

```diff
@@ -572,6 +572,119 @@ var _ = SIGDescribe("Hotplug", func() {
 		)
 	})
 
+	Context("Offline VM with a block volume", func() {
+		var (
+			vm *v1.VirtualMachine
+			sc string
+		)
+
+		BeforeEach(func() {
+			var exists bool
+
+			sc, exists = libstorage.GetRWXBlockStorageClass()
+			if !exists {
+				Skip("Skip test when RWXBlock storage class is not present")
+			}
+
+			vmi, _ := tests.NewRandomVirtualMachineInstanceWithBlockDisk(
```

Comment:
As is this will result in more boilerplate (no libvmi variant with persistent disk encapsulated in)

Reply:
```
libvmi.New(
	libvmi.WithDataVolume("disk0", "missing-datavolume"),
	libvmi.WithResourceMemory("128Mi"),
	WithCloudInitNoCloudUserData("#!/bin/bash\necho hello\n", true)
)
````
Imho this tells me more...

I see that you would also need to create the datavolume, at least it would be visible.

Reply:
So we don't actually use this other than this one occurrence
```bash
$ git grep libvmi.WithDataVolume | wc -l
1
```
Which says a lot;
I think we can follow up on this by with a PR that gets rid of NewRandomVirtualMachineInstanceWithBlockDisk and introduces a proper libvmi permutation